feat: added utils/every-in-by (#1409)
stdlib-js/stdlib: lib/node_modules/@stdlib/utils/every-in-by/docs/repl.txt

{{alias}}( object, predicate[, thisArg ] )

>     Tests whether all elements in a object pass a test implemented by a
>     predicate function, iterating from right to left.

Reviewer: ```suggestion
    Test whether all properties (own and inherited) of an object pass a 
    test implemented by a predicate function.
```

    The predicate function is provided three arguments:

    - `value`: object value
    - `key`: object key
    - `object`: the input object

    The function immediately returns upon encountering a non-truthy return
    value.

    If provided an empty object, the function returns `true`.

    Parameters
    ----------
    object: Object
        Input object over which to iterate.

    predicate: Function

>         The test function.

Reviewer: ```suggestion
        Test function.
```
